docs/polymorphism.md

```diff
@@ -34,6 +34,13 @@ import kotlinx.serialization.*
 import kotlinx.serialization.json.*
 -->
 
+<!--- INCLUDE .*-poly-18.*
```

Comment:
It's fragile (will need to fixed during any renumbering). Don't put the file name here. Just write INCLUDE directive without params to include this piece of code into the next example. And place it after the opening header of the section to help future editors of this text/code finding it.